packages/nimble-components/src/table-column/number-text/index.ts

```diff
@@ -107,11 +126,20 @@ export class TableColumnNumberText extends TableColumnTextBase {
     }
 
     private createFormatter(): NumberFormatter {
+        let minimumDigits;
+        let maximumDigits;
         switch (this.format) {
             case NumberTextFormat.decimal:
+                minimumDigits = typeof this.decimalMaximumDigits === 'number'
+                    ? 0
+                    : this.decimalDigits ?? defaultDecimalDigits;
+                maximumDigits = this.decimalMaximumDigits
```

Comment:
Can't these just be const?
```suggestion
                const minimumDigits = typeof this.decimalMaximumDigits === 'number'
                    ? 0
                    : this.decimalDigits ?? defaultDecimalDigits;
                const maximumDigits = this.decimalMaximumDigits
```

Reply:
No, it results in this eslint violation: https://eslint.org/docs/latest/rules/no-case-declarations

Reply:
@m-akinc re-opening as the linked doc shows the correct implementation. Surround it in a block.

![image](https://github.com/ni/nimble/assets/1588923/bece10d9-5de4-4e14-af9f-d16169dd5f1f)

Reply:
Duh, of course. This was re-opened after the PR was merged, so I'll add this change to my next PR.